app/logic/Mail/OWA/OWAAccount.ts

```diff
@@ -70,7 +73,7 @@ export class OWAAccount extends MailAccount {
   }
 
   async verifyLogin(): Promise<void> {
-    await this.loginCommon(true);
+    await this.loginCommon(false);
```

Comment:
This has to be interactive, because it's the very first time we log in, and the user is there, and it *is* in fact interactive.

Reply:
`startLogin()` is in the UI and irrelevant here. Whatever the UI does is always - by definition - irrelevant to logic. Logic works independent of the UI.

`verifyLogin()` is inherently interactive.